docs/guide/snaps-development-guide.md

```diff
@@ -14,7 +14,15 @@ Read on to learn more!
 ## The Snaps CLI
 
 Before continuing, you should know that [`@metamask/snaps-cli`](https://www.npmjs.com/package/@metamask/snaps-cli) exists, and will be one of your most important tools as you get started with snap development.
-The CLI can be installed globally using `npm` or `yarn`, and provides commands for initiating a snap project and building, executing, and serving your snap for local development.
+The CLI can be installed globally using `npm` or `yarn`:
+
+```sh
+yarn global add @metamask/snaps-cli
+# or...
+npm i -g @metamask/snaps-cli
+```
```

Comment:
Local installation makes it easier for other developers working on the same project to use the CLI (through scripts), so I think we should prefer that over global installation.

Reply:
If we want to recommend that people use the CLI to create new projects, how would they do that?

Reply:
How about having a separate package `create-metamask-app` that can [work with `yarn create`](https://classic.yarnpkg.com/blog/2017/05/12/introducing-yarn/), this would be the equivalent of the current `mm-snap init`. And then `mm-snap` is a local install that just deals with the management of the current snap?

Reply:
I like that idea @ziad-saab.

In the meantime, given that this seems to be the current process, maybe it's okay to merge this as-is?